c/test.py

```diff
@@ -1,97 +1,98 @@
-#! /usr/bin/env python3
```

Comment:
I think this whole file got Windows-newline-ified, maybe unintentionally?

Reply:
Yes, looks like it was unintentional from in https://github.com/BLAKE3-team/BLAKE3/pull/472/commits/26738cd43a7360311596d4a8cb1857138234b9c5

Reply:
Should be fixed now.